acceptance/pages/editor_app.rb

```diff
@@ -40,4 +39,13 @@ def set_component_type_field(value)
     page.execute_script("document.getElementById('page_component_type').value = '#{value}'")
   end
 
+  def submit_new_page
+    #find('#new_page input[type="submit"]', visible: false)
```

Comment:
do we need to keep this in?

Reply:
No, I should have deleted when replacement line was working. Will remove.